src/booking/BookingForm.js

```diff
@@ -0,0 +1,60 @@
+// The Booking Form allows clients that have purchased packages to book a time with the accountant
+// The Booking form requests a booking time and a description of the booking
+
+import React from 'react';
+import {Form, Button} from 'react-bootstrap';
+import './BookingForm.css'
+
+class BookingForm extends React.Component {
+  constructor(props) {
+    super(props);
+    // assuming that the value of the available booking times are structured in an array format
+    // retrieved from the API (i.e: ['Monday at 4:00PM', 'Tuesday at 3:00PM'])
```

Comment:
I'm thinking that the available times will be an array of objects, such as
{
  monday: ['8am': true, '9am': false, ...],
  tuesday: ['8am': false', '9am': true, ...],
  ...
}

But let's go over this tomorrow, so don't change anything now